pack/Microsoft.Azure.Functions.PythonWorker.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.Azure.Functions.PythonWorker</id>
+    <version>1.0.0</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Microsoft Azure Functions Python Worker</description>
+    <copyright>© .NET Foundation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <file src="..\3.6_WINDOWS_x64\**" target="tools\3.6\WINDOWS\X64" />
+    <!-- <file src="..\3.6_WINDOWS_X86\**" target="tools\3.6\WINDOWS\X86" /> -->
+    <file src="..\3.6_LINUX_X64\**" target="tools\3.6\LINUX\X64" />
+    <file src="..\3.6_OSX_X64\**" target="tools\3.6\OSX\X64" />
+    <file src="..\3.7_WINDOWS_X64\**" target="tools\3.7\WINDOWS\X64" />
+    <file src="..\3.7_WINDOWS_X86\**" target="tools\3.7\WINDOWS\X86" />
+    <file src="..\3.7_LINUX_X64\**" target="tools\3.7\LINUX\X64" />
```

Comment:
@maiqbal11, looks like https://github.com/Azure/azure-functions-host/pull/4985/files#diff-41f6f4a743bf773c5edaa3da72f2df34R28 is not all upper case. It may cause issues as Linux file system is case sensitive.

cc: @Hazhzeng

Reply:
The `OSPlatform.ToString()` method is documented to be upper case: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.osplatform.tostring?view=netcore-3.0. We should be fine here.

Reply:
Oh perfect! Thanks for clarifying.